actions/issue_create.yaml

```diff
@@ -28,7 +28,7 @@ parameters:
     position: 3
   labels:
     description: "The comma-separated list of labels"
```

Comment:
Idem, `An array of labels` plutôt ? (genre que les gens mettent pas des virgules)